public/pages/CreateMonitor/components/MonitorExpressions/expressions/__snapshots__/WhereExpression.test.js.snap

```diff
@@ -20,7 +20,7 @@ exports[`WhereExpression renders 1`] = `
       <span
         class="euiExpression__value"
       >
-        all fields are included
+        Select a field
```

Comment:
Seems to be undoing the change from Mihir's PR:
https://github.com/opendistro-for-elasticsearch/alerting-kibana-plugin/pull/54/files

Reply:
Thanks, will merge it

Reply:
I think I replaced at wrong place. This should be fixed with new change.

Reply:
Published a new change, please help review